FalastinShop.SIMS/DB/AppDB.cs

```diff
@@ -0,0 +1,197 @@
+﻿using FalastinShop.SIMS.ProductManagment;
+using System.Data.SqlClient;
+namespace FalastinShop.SIMS.DB;
+
+public class AppDB : IAppDB
+{
+    public static readonly string connectionStr = @"Server=WX1094183;Database=Inventory;Trusted_Connection=True;";
```

Comment:
this is not the best way to do that, use appsettings.json file or any other file to save the connections string